gtsfm/utils/triangulation.py

```diff
@@ -0,0 +1,47 @@
+"""Utilities to support triangulation.
+
+Authors: Ayush Baid
+"""
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler
+
+
+def calculate_triangulation_angle_in_degrees(
```

Comment:
can we cite COLMAP's equivalent fn in the docstring? https://github.com/colmap/colmap/blob/dev/src/base/triangulation.cc#L122